yb-voyager/cmd/importData.go

```diff
@@ -371,8 +371,8 @@ func generateSmallerSplits(taskQueue chan *SplitFileImportTask) {
 	}
 
 	excludeTableList := utils.CsvStringToSlice(target.ExcludeTableList)
-	importTables = removeExcludeTables(importTables, excludeTableList)
-	allTables = removeExcludeTables(allTables, excludeTableList)
+	importTables = utils.RemoveExcludeList(importTables, excludeTableList)
```

Comment:
Should we rename `RemoveExcludeList` as `SetDifference`?

Reply:
Sounds good to me, I saw the opportunity to re-use code and took it!